src/pages/AssetsPage.vue

```diff
@@ -52,9 +52,9 @@ export default defineComponent({
 </script>
 <style lang="scss" scoped>
 #assets-top {
-  padding: 0 16px;
```

Comment:
Could you advise why deleted here for mobile devices? @Kahonnohak @niklabh It makes the divider size longer.

<img width="462" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/92044428/1d2a733d-b570-4c86-9193-327c919b349e">

Reply:
@sirius651

Reply:
@sirius651  Could you fix it?

Reply:
@sirius651 or you can fix it in [this](https://github.com/AstarNetwork/astar-apps/pull/803) PR.

Reply:
@sirius651 I fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/803/commits/486415b6caf7ecf2548f97d75bad837ac2b2ba47) commit.